CMakeLists.txt

```diff
@@ -40,12 +40,6 @@ foreach(pold "") # Currently Empty
     endif()
 endforeach()
 
-# Build the library with C++11 standard support, independent from other including
-# software which may use a different CXX_STANDARD or CMAKE_CXX_STANDARD.
-set(CMAKE_CXX_STANDARD 11)
-set(CMAKE_CXX_EXTENSIONS OFF)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
Removing this seems like another change of behavior.  Why are these removed here?

Reply:
No, super discouraged to use the global rules, they produce the overwriting at global level for all the TARGETS defined in this project and those who want to include jsoncpp as a dependency, instead the own definition is used to each TARGET individually.

```cmake
set_target_properties(jsoncpp.framework PROPERTIES CXX_STANDARD 11)
set_target_properties(jsoncpp.framework PROPERTIES CXX_EXTENSIONS OFF)
set_target_properties(jsoncpp.framework PROPERTIES CXX_STANDARD_REQUIRED ON)
```